### ISSUE TITLE
FluidDynamics Warning in Appveyor

### ISSUE THREAD

**Issue by philbucher** [KratosMultiphysics/Kratos#1589]
Hi @KratosMultiphysics/fluid-dynamics 
I was going through the Appveyor output and noticed the following warning, I don't know if it is relevant or not:
~~~
[00:22:52] c:\projects\kratos\applications\fluiddynamicsapplication\custom_utilities\element_size_calculator.h(99): warning C4661: 'double Kratos::ElementSizeCalculator<2,4>::GradientsElementSize(const boost::numeric::ublas::bounded_matrix<double,4,2,boost::numeric::ublas::row_major> &)': no suitable definition provided for explicit template instantiation request [C:\projects\kratos\cmake_build\applications\FluidDynamicsApplication\KratosFluidDynamicsApplication.vcxproj]
[00:22:53]   c:\projects\kratos\applications\fluiddynamicsapplication\custom_utilities\element_size_calculator.h(95): note: see declaration of 'Kratos::ElementSizeCalculator<2,4>::GradientsElementSize'
[00:22:53] c:\projects\kratos\applications\fluiddynamicsapplication\custom_utilities\element_size_calculator.h(99): warning C4661: 'double Kratos::ElementSizeCalculator<3,8>::GradientsElementSize(const boost::numeric::ublas::bounded_matrix<double,8,3,boost::numeric::ublas::row_major> &)': no suitable definition provided for explicit template instantiation request [C:\projects\kratos\cmake_build\applications\FluidDynamicsApplication\KratosFluidDynamicsApplication.vcxproj]
[00:22:53]   nanoparticle.cpp
[00:22:54]   c:\projects\kratos\applications\fluiddynamicsapplication\custom_utilities\element_size_calculator.h(95): note: see declaration of 'Kratos::ElementSizeCalculator<3,8>::GradientsElementSize'
[00:22:54] C:\projects\kratos\applications\FluidDynamicsApplication\custom_utilities\element_size_calculator.cpp(18): warning C4661: 'double Kratos::ElementSizeCalculator<2,4>::GradientsElementSize(const boost::numeric::ublas::bounded_matrix<double,4,2,boost::numeric::ublas::row_major> &)': no suitable definition provided for explicit template instantiation request [C:\projects\kratos\cmake_build\applications\FluidDynamicsApplication\KratosFluidDynamicsApplication.vcxproj]
[00:22:56]   c:\projects\kratos\applications\fluiddynamicsapplication\custom_utilities\element_size_calculator.h(95): note: see declaration of 'Kratos::ElementSizeCalculator<2,4>::GradientsElementSize'
[00:22:56] C:\projects\kratos\applications\FluidDynamicsApplication\custom_utilities\element_size_calculator.cpp(18): warning C4661: 'double Kratos::ElementSizeCalculator<3,8>::GradientsElementSize(const boost::numeric::ublas::bounded_matrix<double,8,3,boost::numeric::ublas::row_major> &)': no suitable definition provided for explicit template instantiation request [C:\projects\kratos\cmake_build\applications\FluidDynamicsApplication\KratosFluidDynamicsApplication.vcxproj]
[00:22:57]   c:\projects\kratos\applications\fluiddynamicsapplication\custom_utilities\element_size_calculator.h(95): note: see declaration of 'Kratos::ElementSizeCalculator<3,8>::GradientsElementSize'
[00:22:57]   vms.cpp
~~~

**Comment by jcotela** [comment]
The issue is known, we decided not to add the "missing" instantiations to ensure we get a compile error if one tries to use it with wrong geometries. I will try to think a way to get the same "functionality" without the warning though...

**Comment by jcotela** [comment]
Fixed in #1594
